docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -102,19 +135,35 @@ PSOutput Main(PSInput input)
 
 Here as well we have an input `PSInput`, and an output `PSOutput`.
 
-Since there aren't any other shaders in between the VS and the PS, the VS's output is the PS's input, the naming might be a bit confusing but that's the gist of it, PSInput should match the VSOutput in vertex shader, this isn't entirely required but not doing so is only advisable if you really know what you are doing.
+Since there aren't any other shaders in between the VS and the PS, the VS's 
+output is the PS's input, the naming might be a bit confusing but that's the 
+gist of it, PSInput should match the VSOutput in vertex shader, this isn't 
+entirely required but not doing so is only advisable if you really know what 
+you are doing.
 
-Next we have our output, `D3D11` gives us the possibility to write to multiple render targets, but we are not doing that, so we'll just be writing a `float4` as our output, which is an RGBA color.
 
-Notice how we have another semantic string attached to the `color` field, this semantic string specifies which render target we want to be writing to, the `0` after `SV_Target` is the index of our render target, in our case, we have only one, so we write `SV_Target0` or `SV_Target`.
+Next we have our output, `D3D11` gives us the possibility to write to multiple 
+render targets, but we are not doing that, so we'll just be writing a `float4` 
+as our output, which is an RGBA color.
 
-`D3D11` lets us write up to 8 render targets simultaneously from the same pixel shader, those come in handy when implementing more advanced shading techniques, for example a popular technique that uses 4 or more
+Notice how we have another semantic string attached to the `color` field, 
+this semantic string specifies which render target we want to be writing to, 
+the `0` after `SV_Target` is the index of our render target, in our case, 
+we have only one, so we write `SV_Target0` or `SV_Target`.
 
-And lastly, our `Main` function, following the same pattern as in the VS, we have one parameter, the input, and one return value, the output, again we create an instance of `PSOutput`, initialize everything to 0, and write the color we got from the input, to our output.
+`D3D11` lets us write up to 8 render targets simultaneously from the same pixel 
+shader, those come in handy when implementing more advanced shading techniques, 
+for example a popular technique that uses 4 or more
```

Comment:
> for example a popular technique that uses 4 or more

too vague, better to remove this entire sentence as we're not diving into deferred rendering (which I assume this is referring to?), which also doesn't specifically require 4 or more

Reply:
yeah... I don't see why that sentence is there... its not even vague - it is incomplete